en/config/actuators.md

```diff
@@ -140,32 +140,64 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
   - The setting limits the rate of change of an actuator (if not specified then no rate limit is applied).
      It is intended for actuators that may be damaged if they move too fast — such as the tilting actuators on a tiltrotor VTOL vehicle.
   - For example, a setting of 2.0 means that the motor/servo will not be commanded to move from 0 to 1 at a rate that completes the operation in less than 2 seconds (in case of reversible motors, the range is -1 to 1).
-- `Lock control surfaces in hover`:
+- (Advanced) `Flap Scale`: How much this actuator is deflected at the "full flaps configuration" [0, 1] (see [Flap Scale and Spoiler Scale Configuration](#flap-scale-and-spoiler-scale-configuration) below).
+  Can be used to configure aerodynamic surface as flap or to compensate for generated torque through main flaps.
+- (Advanced) `Spoiler Scale`: How much this actuator is deflected at the "full spoiler configuration" [0, 1] (see [Flap Scale and Spoiler Scale Configuration](#flap-scale-and-spoiler-scale-configuration) below).
+  Can be used to configure aerodynamic surface as spoiler or to compensate for generated torque through main spoiler.
+- (VTOL only) `Lock control surfaces in hover`:
   - `Enabled`: Most vehicles do not use control surfaces in hover. Use this setting to lock them so that they don't affect vehicle dynamics.
   - `Disabled`: Set this for vehicles that use control surfaces in hover, such as the duo tailsitter (which uses elevons for pitch and yaw control). It should also be set for vehicles that use control surfaces to provide additional stabilization in hover mode when moving at speed or in high winds.
 
+#### Flap Scale and Spoiler Scale Configuration
+
+"Flap-control" and "Spoiler-control" are aerodynamic configurations that can either be commanded manually by the pilot (using RC, say), or are set automatically by the controller.
+For example, a  pilot or the landing system might engage "Spoiler-control" in order to reduce the airspeed before landing.
+
+The configurations are an _abstract_ way for the controller to tell the allocator how much it should adjust the aerodynamic properties of the wings relative to the "full flaps" or "full spoiler" configuration (between `[0,1]`, where "1" indicates the full range).
+The allocator then uses any of the available control surfaces it wants in order to achieve the requested configuration: usually flaps, ailerons, and elevator.
+
+The `flap scale` and `spoiler scale` settings in the actuator UI inform the allocator how much ailerons, elevators, flaps, spoilers, and other control surfaces, contribute to a requested "Flap-control" and/or "Spoiler-control" value.
+Specifically, they indicate how much each control surface should be deflected when the controller is demanding "full flaps" or "full spoiler".
+
+In the following example, the vehicle has two ailerons, one elevator, one rudder and two flaps as control surfaces:
```

Comment:
@sfuhrer I have re-written this section in line with your confirmation, mostly just chopping and changing your words.
I think it is clear now?

It would be useful to state how you decide that say, a `flap scale` of .3 is reasonable, but perhaps we should merge this and discuss that as a post process?

Reply:
Yes your rewording makes it much clearer, thank you!

And yes I would go ahead and merge it. Setting the appropriate scales is mostly an iterative process based on flight data, we could add this comment if you think it helps.

Reply:
@sfuhrer Thanks very much - this was a hard one!

>Setting the appropriate scales is mostly an iterative process based on flight data, we could add this comment if you think it helps.

I'm in two minds - on one hand it would mean people would know we thought about it and can't provide this - that's good because it reduces some sorts of questions.

The problem is, people are still stuck. This is really nothing you could provide a "script" on, or guidance for reading the log? I'm thinking might be an interesting dev conference presentation for how you approach this on a couple of slightly different vehicles.

Reply:
I've never fine tuned a fixed-wing runway vehicle, so I'm probably not the expert you think I am ;) I just enable the options and hope they're intuitively enough to follow. 
@tstastny tagging you as FW maintainer, something to maybe keep in mind to add docs at some point.

Reply:
Thanks! @tstastny , I'm watching you :-)